tools/k230/Config.mk

```diff
@@ -0,0 +1,39 @@
+############################################################################
+# tools/k230/Config.mk
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+# These are the macros that will be used in the NuttX make system to compile
+# and assembly source files and to insert the resulting object files into an
+# archive.  These replace the default definitions at tools/Config.mk
+
+# POSTBUILD -- Perform post build operations
+
+# kernel build warning ROMFS lacking case
+ifeq ($(CONFIG_BUILD_KERNEL),y)
+ifeq ($(wildcard $(TOPDIR)/arch/risc-v/src/board/romfs_boot.c),)
+define POSTBUILD
```

Comment:
why not move into Make.defs

Reply:
I can't get you, can you elobarate?

The purpose is to show a POSTBUILD message to warn that the build with stub ROMFS is not for trying on target. Later when rebuild happens with real ROMFS, another POSTBUILD message will confirm that we can move ahead to try on target. 

I saw existing POSTBUILD definitions are mostly in `tools/*/Config.mk`. Did you mean that we should put POSTBUILD definitions in "Make.defs"? 

I guess the $(wildcard) here can be replaced with something like "K230_ROMFS_STUB" though, which I haven't tried yet. If you mean this, then I think I could try harder.

Reply:
Other boards put Config.mk into tools/*/Config.mk, because the folders contain additional tools to do some special work.
But in your simple case, it's more suitable to define POST_BUILD in https://github.com/apache/nuttx/pull/11489/files#diff-f92b47a398e385fa574860609a99076bac7a71f13403db16b4a08253a9515b37 directly.
BTW, why not fix romfs issue directly? @fxysunshine created a patch fixed the romfs issue in kernel/protected mode recently. @fxysunshine could you upstream the patch?

Reply:
@yf13  a fixed change romfs mount that moves from nsh to sched/init, that is compatible for kernel/protected mode, please reference.
https://github.com/apache/nuttx/pull/11498

Reply:
@xiaoxiang781216 and @fxysunshine thanks.

I don't know that POSTBUILD can be added to board level Make.defs or Makefile as I haven't found any existing uses. If this is the right direction, I can try it. But please teach under what conditions that "tools/.../Config.mk" is a better fit?

As for the ROMFS of "canmv230/knsh", it is mainly the user land apps like "/system/bin/init" and "/system/bin/ostest" etc. That ROMFS is normally generated by "../apps/tooks/mkromfs.sh".  The etc ROMFs is not in use right now.  So I don't know how to reuse patch from @fxysunshine.

Reply:
I think We don't need export ROMFS image to userland apps like "/system/bin/init", that is complicated in kernel build. We can custom "etc" name to other and unify the ROMFS for kernel build and flat build.

Reply:
@xiaoxiang781216 POSTBUILD actions have been moved to boards level `Make.defs`. I hope it is the right place for add above mentioned post-processing stuffs later.

@fxysunshine I am not sure what you mean by "export ROMFS image to userland apps", per my understanding, the apps in the ROMFS doesn't know anything about where they stay. The ROMFS I created is mounted by "nx_bringup" directly and I only register the "/dev/ram0" in `board_late_initialize()`. If you are saying about a new ROMFS design that can easily have initramfs (including all kinds of programs, databases, scripts etc) without extra programming, it is nice and I can learn it later. :)